components/repository-validator/base/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - validating-admission-policy.yaml
+  - validating-admission-policy-binding.yaml
+
+namespace: repository-validator
```

Comment:
this is causing to apply a NS on the cluster scoped resources, i.e. ValidatingAdmissionPolicy and ValidatingAdmissionPolicyBinding. Remove that from here and from the kustomization files in staging folder.

Reply:
Good catch, fixed